src/Data/Product/Properties.agda

```diff
@@ -23,7 +25,17 @@ module _ {a b} {A : Set a} {B : A → Set b} where
   ,-injectiveˡ : ∀ {a c} {b : B a} {d : B c} → (a , b) ≡ (c , d) → a ≡ c
   ,-injectiveˡ refl = refl
 
-  -- See also Data.Product.Properties.WithK.,-injectiveʳ.
+  ,-injective≡ʳ : ∀ {a b} {c : B a} {d : B b} → UIP A → (a , c) ≡ (b , d) → (q : a ≡ b) → subst B q c ≡ d
```

Comment:
I think it would be more consistent with the naming conventions in the library to call this one `,-injectiveʳ-≡` and the one below `,-injectiveʳ-UIP`